test/integration/smoke-testing/test_signing_in_and_my_stuff.js

```diff
@@ -5,34 +5,19 @@
  *
  */
 
+var selenium_helpers = require('../../helpers/selenium-helpers.js');
+const clickText = selenium_helpers.clickText;
+const clickXpath = selenium_helpers.clickXpath;
+const findByXpath = selenium_helpers.findByXpath;
+const clickButton = selenium_helpers.clickButton;
+const driver = selenium_helpers.driver;
```

Comment:
As long as you are using object destructuring [here](https://github.com/LLK/scratch-www/pull/1458/files#diff-b6f5abc4f1954011142c27f29c582857R7), you may as well simplify these requires like,

```javascript
const {
    clickText,
    clickXpath,
    findByXpath,
    clickButton,
    driver
} = require('../../helpers/selenium-helpers.js');
```